### DISCUSSION TITLE
Device locatie naam / omschrijving komen niet beide voor in de API?

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#278] [Q&A] [answered]
Wanneer ik via https://demo.sensorenregister.nl/api/device?legalEntityId=61267b03-9024-4e90-9366-c0eebaed1123 de actuele stand van mijn organisatie uit het register op wil halen,  lijkt het alsof de naam + beschrijving van een locatie nooit beide gevuld zijn. Ook niet als dat in de UI wel zo is. 

Description lijkt alleen aanwezig als er geen naam gevuld is. Is er wel een naam gevuld, dan is description niet aanwezig in de response. Weten jullie misschien waar dit aan ligt?

Voorbeeld stukje uit de response:

   "locationDetails":{
      "name":"OudeStadsgracht-designhuis"
   }

(ID device: 25a23d9f-1f67-49e4-92bf-619b9c8d24d8)

**Comment by kad-floriw** [comment]
Ik kan het niet reproduceren! Bestaat de organisatie nog? Ik zie hem hier niet tussen staan: https://demo.test.sensorenregister.nl/organization. Ik had het even getest voor een andere organisatie, en daar werkt het wel.

**Reply by CelineJansen** [reply]
Hoi Wim, Op jouw link kan ik niet inloggen... Misschien kijken we op een andere omgeving?

![image](https://user-images.githubusercontent.com/79267472/146392468-05bfde78-d30e-458e-8860-7c9603ccc7e6.png)

**Comment by CelineJansen** [comment]
Er ligt iets anders aan ten grondslag volgens mij.
Wanneer ik het probeer te reproduceren door ergens zowel naam als beschrijving te vullen in de tool doet hij het volgende:
[15:57 - er staat alleen een beschrijving]
![image](https://user-images.githubusercontent.com/79267472/146395032-7541c7ae-50f4-4e28-8858-ec0eab75f8cf.png)

[15:58 - naam toegevoegd en opgeslagen]
![image](https://user-images.githubusercontent.com/79267472/146395187-d2a7bc93-c4a3-469f-bd98-047fddc6119a.png)

Als ik dan terug ga door op "apparaten" te klikken, en dezelfde device weer zoek. Staat er alleen nog maar een naam. De beschrijving is dan verdwenen.
![image](https://user-images.githubusercontent.com/79267472/146395484-9cd37c68-f902-4f81-a6c9-6677b05cbcbc.png)

Logisch dat ik dan in de API alleen een naam terug krijg. Niet zo logisch dat mijn beschrijving verdwijnt (?)

**Reply by kad-floriw** [reply]
Yes inderdaad, als de velden al eens ingevuld lijkt er nu 1 van de 2 te verdwijnen bij het opslaan (maar niet als de velden nog niet zijn ingevuld). Ik ga het oplossen!

**Comment by marcvanandel** [comment]
Ik heb hier een bug (issue) voor aangemaakt, want dat is wat het is, right? -> #280

**Comment by kad-floriw** [comment, marked as answer]
Inmiddels is de code gemerged, thanks voor het doorgeven! Binnenkort zal het live gezet worden op de omgevingen.

**Reply by marcvanandel** [reply]
Is de week van 10 januari snel genoeg om dit in de omgevingen te krijgen, @CelineJansen ? (voelt wat ver weg ... maar waarschijnlijk hebben we allemaal / velen van ons toch vakantie? 😁 )

**Reply by CelineJansen** [reply]
Hoi @marcvanandel , ja hoor wat mij betreft helemaal prima. Fijn dat jullie het op gaan pakken. Prettige feestdagen!

**Reply by kad-busses** [reply]
De demo-omgeving is geupdate 🙂

**Reply by marcvanandel** [reply]
Dank je, @kad-busses !! 💯
